src/classes/Bucket.ts

```diff
@@ -234,7 +234,7 @@ export class Bucket {
   }
 
   /**
-   * checks a lender's LP balance in a bucket
+   * Checks a lender's LP balance in a bucket.
```

Comment:
nit: `Retrieves` would be more accurate than `Checks`

Reply:
Addressed in 4dc9c72f655776daa1b9a36f70526e836760235f